src/_C014.ino

```diff
@@ -67,872 +68,1069 @@ bool CPlugin_014_sendMQTTmsg(String& topic, const char* payload, int& errorCount
+                log += F(" / ");
+                log += ExtraTaskSettings.TaskDeviceValueNames[event->Par2-1];
+                log += F(" sensorType:");
+                log += event->sensorType;
+                log += F(" Source:");
+                log += event->Source;
+                log += F(" idx:");
+                log += event->idx;
+                log += F(" S1:");
+                log += event->String1;
+                log += F(" S2:");
+                log += event->String2;
+                log += F(" S3:");
+                log += event->String3;
+                log += F(" S4:");
+                log += event->String4;
+                log += F(" S5:");
+                log += event->String5;
+                log += F(" P1:");
+                log += event->Par1;
+                log += F(" P2:");
+                log += event->Par2;
+                log += F(" P3:");
+                log += event->Par3;
+                log += F(" P4:");
+                log += event->Par4;
+                log += F(" P5:");
+                log += event->Par5;
+                addLog(LOG_LEVEL_DEBUG, log);
+              } */
+      success = false;
+
+      if (!string.isEmpty()) {
+        String commandName = parseString(string, 1); // could not find a way to get the command out of the event structure.
+
+        if (commandName == F("gpio"))                // !ToDo : As gpio is like any other plugin commands should be integrated below!
+        {
+          int port         = event->Par1;            // parseString(string, 2).toInt();
+          int valueInt     = event->Par2;            // parseString(string, 3).toInt();
+          String valueBool = F("false");
+
+          if (valueInt == 1) { valueBool = F("true"); }
+
+          String topic = CPLUGIN_014_PUBLISH; // ControllerSettings.Publish not used because it can be modified by the user!
+          topic.replace(F("%sysname%"), Settings.Name);
+          topic.replace(F("%tskname%"), F(CPLUGIN_014_SYSTEM_DEVICE));
+          topic.replace(F("%valname%"), String(F(CPLUGIN_014_GPIO_VALUE)) + toString(port, 0));
+
+          success = MQTTpublish(CPLUGIN_ID_014, INVALID_TASK_INDEX, topic.c_str(), valueBool.c_str(), false);
+
+          if (loglevelActiveFor(LOG_LEVEL_INFO) && success) {
+            String log = F("C014 : Acknowledged GPIO");
+            log += port;
+            log += F(" value:");
+            log += valueBool;
+            log += F(" (");
+            log += valueInt;
+            log += F(")");
+            log += F(" success!");
+            addLog(LOG_LEVEL_INFO, log);
+          }
+
+          if (loglevelActiveFor(LOG_LEVEL_ERROR) && !success) {
+            String log = F("C014 : Acknowledged GPIO");
+            log += port;
+            log += F(" value:");
+            log += valueBool;
+            log += F(" (");
+            log += valueInt;
+            log += F(")");
+            log += F(" ERROR!");
```

Comment:
Ditto.

Reply:
One disadvantage of merging strings is that it may result in unique strings which cannot be used by the linker in the deduplicate call.
Not sure these are unique, but worth a try to check.